app/routes/applications.js

```diff
@@ -114,19 +114,37 @@ module.exports = router => {
     })
   })
 
-  router.get('/applications/:id/school-placement', (req, res) => {
+  router.get('/applications/:id/school-placement(-second)?', (req, res) => {
+    console.log(req.params);
     const { id } = req.params
+    var priority = req.params[0]
+    if ( priority) {
+      priority = priority.substring(1)
+    } else {
+      priority = 'first'
+    }
 
-    const placementItems = data.placements
+    var placementItems = data.placements
```

Comment:
Per my previous comment, `var placementItems = data.placements` should be `let placementItems = data.placements` i.e., replace `var` with `let`.

Reply:
Thanks @simonwhatley! Resolved in https://github.com/DFE-Digital/apply-for-teacher-training-prototype/pull/871